src/Explorer/Explorer.ts

```diff
@@ -107,6 +106,12 @@ interface AdHocAccessData {
   readUrl: string;
 }
 
+export interface ExplorerParams {
+  setIsNotificationConsoleExpanded: (isExpanded: boolean) => void;
```

Comment:
Same as `setInProgressMessageIdToBeDeleted`, I would like to keep this name since it's a hook and the actual functions that we expose in Explorer are called `expandConsole` and `collapseConsole`.